### PR TITLE
Fix compilation error. Use new IO API

### PR DESCRIPTION
**Pull request by DomGarguilo**
The build was failing with the following:
```
Compilation failure: 
[ERROR] /home/runner/work/accumulo-testing/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/bulk/SelectiveQueueing.java:[24,53] package org.apache.accumulo.core.client.admin.servers does not exist
[ERROR] /home/runner/work/accumulo-testing/accumulo-testing/src/main/java/org/apache/accumulo/testing/performance/tests/SplitBalancingPT.java:[28,53] package org.apache.accumulo.core.client.admin.servers does not exist
```

### CODE REVIEW

File: src/main/java/org/apache/accumulo/testing/performance/tests/SplitBalancingPT.java [hunk 1]
```diff
@@ -61,8 +60,7 @@ public Report runTest(final Environment env) throws Exception {
     client.tableOperations().addSplits(TABLE_NAME, getSplits());
     client.instanceOperations().waitForBalance();
 
-    int totalTabletServers =
-        client.instanceOperations().getServers(ServerId.Type.TABLET_SERVER).size();
+    int totalTabletServers = client.instanceOperations().getTabletServers().size();
```
Comment (keith-turner) [inline src/main/java/org/apache/accumulo/testing/performance/tests/SplitBalancingPT.java:63]: This getTabletServers() method is currenty deprecated in main and using it here will cause the build to fail.  So we should not be going back to the deprecated code.  Not sure why but it seems the gh actions build is using an older version of accumulo 4.0-snap.

Locally when I do the following the accumulo-testing build will fail.

```
git checkout  main
git pull
mvn clean install -PskipQA
 cd ../accumulo-testing/
git checkout fixBuild
mvn clean package -DskipTests
```

Will see the following errors.

```
[INFO] --- compiler:3.11.0:compile (default-compile) @ accumulo-testing ---
[INFO] Changes detected - recompiling the module! :source
[INFO] Compiling 207 source files with javac [debug deprecation release 11] to target/classes
[INFO] -------------------------------------------------------------
[WARNING] COMPILATION WARNING : 
[INFO] -------------------------------------------------------------
[WARNING] accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/bulk/SelectiveQueueing.java:[40,50] getTabletServers() in org.apache.accumulo.core.client.admin.InstanceOperations has been deprecated
[WARNING]accumulo-testing/src/main/java/org/apache/accumulo/testing/performance/tests/SplitBalancingPT.java:[63,57] getTabletServers() in org.apache.accumulo.core.client.admin.InstanceOperations has been deprecated
[INFO] 2 warnings 
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/bulk/SelectiveQueueing.java: warnings found and -Werror specified
[INFO] 1 error

```

Comment (keith-turner) [reply src/main/java/org/apache/accumulo/testing/performance/tests/SplitBalancingPT.java:63]: https://github.com/apache/accumulo/blob/94167151e2de1be87ac60e25751edf1b557d3a3b/core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java#L218

That is a recent commit on main where the method is deprecated.

Comment (keith-turner) [reply src/main/java/org/apache/accumulo/testing/performance/tests/SplitBalancingPT.java:63]: Asked about this on slack 

https://the-asf.slack.com/archives/CERNB8NDC/p1729614591946599?thread_ts=1729614322.448179&cid=CERNB8NDC

Seems the apache jenkins sever has not built accumulo 4.0.0-snapshot and that is causing the mismatch between the latest commit in accumulo/main and the github build.

Comment (keith-turner) [reply src/main/java/org/apache/accumulo/testing/performance/tests/SplitBalancingPT.java:63]: Ran the build on apache jenkins server and produiced a new accumulo 4.0.0-snapshot jar.  Now the latest code is available to accumulo-testing and its building now.

Comment (DomGarguilo) [reply src/main/java/org/apache/accumulo/testing/performance/tests/SplitBalancingPT.java:63]: Looks like the build is passing in the accumulo-testing `elasticity` branch. So maybe this PR should be closed.

**Pull request closed without merging**